### DISCUSSION TITLE
remote access of the logs is it possible ?

### DISCUSSION THREAD

**Discussion by vamsi0303** [COVESA/dlt-daemon#689] [Q&A] [answered]
Can we see the logs generated by an application on machine1 from machine2? If yes, how can I achieve this?

**Comment by michael-methner** [comment, marked as answer]
Hello @vamsi0303 ,
this sounds like DLT multinode feature: https://github.com/COVESA/dlt-daemon/blob/master/doc/dlt_multinode.md
